### DISCUSSION TITLE
Is there a way to move or rotate 3D elements individually?

### DISCUSSION THREAD

**Discussion by Myzik-Dev** [iTwin/itwinjs-core#2076] [Q&A] [answered]
Is there a way to move or rotate 3D elements individually?

if there is a way, I would appreciate it if you could let me know.

**Comment by pmconne** [comment, marked as answer]
The `alpha` [editing packages](https://github.com/imodeljs/imodeljs/tree/master/editor/frontend) include tools for interactively rotating, moving, or otherwise transforming elements and updating their placement in the iModel.
If that doesn't help, please be more specific about what you're trying to accomplish.

**Reply by Myzik-Dev** [reply]
Thank you for your reply.
I'll test it.

**Reply by paprophet** [reply]
does the model  ( create from snapshot )  support tools for interactively rotating, moving, or otherwise transforming elements and updating their placement ?
does save the rotating, moving the element data to the model   (the model create from snapshot   )?

**Reply by kabentley** [reply]
Snapshots, as the name implies, are intended to be readonly. 

Since the only way to coordinate the work of multiple people editing copies of iModels at the same time is to use briefcases, the editing packages presume they're working on briefcases. 

However, for testing, you can use a "standalone" iModel (see [StandaloneDb.createEmpty](https://www.itwinjs.org/reference/imodeljs-backend/imodels/standalonedb/createemptystatic/)) and set `allowEdit: true`. Of course that's not intended for production code, since only one user at a time may edit the iModel.

**Reply by Myzik-Dev** [reply]
Thank you for the detailed explanation :)
